microshift_running_apps/microshift-embedded-apps-on-rhel-edge.adoc

```diff
@@ -0,0 +1,56 @@
+If you have an application that includes APIs, container images, and configuration files for deployment such as manifests, you can build application RPMs. You can then add the RPMs to your {op-system-ostree} system image.
+
+The following is an outline of the procedures to embed applications or workloads in an fully self-contained operating system image:
+
+* Build your own RPM that includes your application manifest.
+* Add the RPM to the blueprint you used to install {product-title}.
+* Add the workload container images to the same blueprint.
+* Create a bootable ISO.
+
+For a step-by-step tutorial about preparing and embedding applications in a {op-system-ostree} image, use the follwoing tutorial:
+
+* xref:../microshift_running_apps/microshift-embedding-apps-tutorial.adoc#microshift-embedding-apps-tutorial[Embedding applications tutorial]
+
+[id="microshift-add-app-manifests-to-image_{context}"]
+== Adding application manifests to an image for offline use
+If you have a simple application that includes a few files for deployment such as manifests, you can add those manifests directly to a {op-system-ostree} system image.
+
+See the "Create a custom file blueprint customization" section of the following {op-system-ostree} documentation for an example:
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/9/html/composing_installing_and_managing_rhel_for_edge_images/composing-a-rhel-for-edge-image-using-image-builder-command-line_composing-installing-managing-rhel-for-edge-images#image-customizations_composing-a-rhel-for-edge-image-using-image-builder-command-line[Create a custom file blueprint customization]
+
+[id="microshift-embed-apps-for-offline-use_{context}"]
+== Embedding applications for offline use
+If you have an application that includes more than a few files, you can embed the application for offline use. See the following procedure:
+
+* xref:../microshift_running_apps/microshift-embed-apps-offline-use.adoc#microshift-embed-apps-offline-use[Embedding applications for offline use]
+
+//additional resources for assembly
+[id="additional-resources_microshift-embed-apps-on-rhel-edge_{context}"]
+[role="_additional-resources"]
+== Additional resources
+* xref:../microshift_install/microshift-embed-in-rpm-ostree.adoc#microshift-embed-in-rpm-ostree[Embedding {product-title} in an RPM-OSTree image]
+
+* link:https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/{op-system-version-major}/html/composing_installing_and_managing_rhel_for_edge_images/index[Composing, installing, and managing {op-system-ostree} images]
+
+* xref:../microshift_install/microshift-embed-in-rpm-ostree.adoc#preparing-for-image-building_microshift-embed-in-rpm-ostree[Preparing for image building]
+
+* link:https://cloud.redhat.com/blog/meet-red-hat-device-edge-with-microshift[Meet Red Hat Device Edge]
+
+* link:https://www.redhat.com/sysadmin/create-rpm-package[How to create a Linux RPM package]
```

Comment:
Given the recent changes to the way Red Hat manages blogs, I am wary about linking to a procedure from the sysadmin blog? Isn't this project abandonware now?

@mwerner2113 WDYT?

Reply:
I want to get this merged, so I'll comment it out for the moment. Easy peasy to add back if we are OK with it.